src/Omega_h_mesh.cpp

```diff
@@ -270,7 +272,7 @@ void Mesh::set_tag(
         topological_plural_name(family(), ent_dim), name.c_str());
   }
   Tag<T>* tag = as<T>(tag_iter(ent_dim, name)->get());
-  OMEGA_H_CHECK(array.size() == nents(ent_dim) * tag->ncomps());
+  //OMEGA_H_CHECK(array.size() == nents(ent_dim) * tag->ncomps());
```

Comment:
Please remove commented code.
This seems like a decent check to make.  Why not keep it?